proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -267,6 +272,150 @@ message SelectOrUpdateRequest {
   repeated string column_specs = 3;
 }
 
+message Expression {
+  oneof type {
+    string column_name = 1;
+    int64 long_value = 2;
+    string raw = 3;
+  }
+}
+
+message Selectable {
+  string column_name = 1;
+  Expression expression = 2;
+}
+
+message Pair {
+  string output_column_name = 1;
+  string input_column_name = 2;
+}
+
+message MathContext {
+  enum RoundingMode {
+    UP = 0;
+    DOWN = 1;
+    CEILING = 2;
+    FLOOR = 3;
+    HALF_UP = 4;
+    HALF_DOWN = 5;
+    HALF_EVEN = 6;
+    UNNECESSARY = 7;
+  }
+  sint32 precision = 1;
+  RoundingMode rounding_mode = 2;
+}
+
+enum BadDataBehavior {
+  // Reset the state for the bucket to {@code null} when invalid data is encountered.
+  RESET = 0;
```

Comment:
Since 0 is the "This is not set" default, consider making it fail-safe with THROW, so as to not surprise users who fail to set this?

Reply:
In context, at least right now, this is always being prefaced w/ `optional`. The idea is that if the client does not provide a value for it, it will inherit the appropriate server defaults (the defaults *vary* depending on the specific field):

```grpc
message UpdateByEmaOptions {
  optional BadDataBehavior on_null_value = 1;
  optional BadDataBehavior on_nan_value = 2;
  ...
```

I'll add documentation at this layer to explain that.

That said, I'm not *against* making `THROW = 0`, but there may be reasons to want the order to be the same as the java enum it is mapping? (We can update the java enum as well...)